rhino/src/main/java/org/mozilla/javascript/Token.java

```diff
@@ -32,205 +32,206 @@ public static enum CommentType {
-            SHEQ = 46, // shallow equality (===)
-            SHNE = 47, // shallow inequality (!==)
-            REGEXP = 48,
-            BINDNAME = 49,
-            THROW = 50,
-            RETHROW = 51, // rethrow caught exception: catch (e if ) use it
-            IN = 52,
-            INSTANCEOF = 53,
-            LOCAL_LOAD = 54,
-            GETVAR = 55,
-            SETVAR = 56,
-            CATCH_SCOPE = 57,
-            ENUM_INIT_KEYS = 58,
-            ENUM_INIT_VALUES = 59,
-            ENUM_INIT_ARRAY = 60,
-            ENUM_INIT_VALUES_IN_ORDER = 61,
-            ENUM_NEXT = 62,
-            ENUM_ID = 63,
-            THISFN = 64,
-            RETURN_RESULT = 65, // to return previously stored return result
-            ARRAYLIT = 66, // array literal
-            OBJECTLIT = 67, // object literal
-            GET_REF = 68, // *reference
-            SET_REF = 69, // *reference    = something
-            DEL_REF = 70, // delete reference
-            REF_CALL = 71, // f(args)    = something or f(args)++
-            REF_SPECIAL = 72, // reference for special properties like __proto
-            YIELD = 73, // JS 1.7 yield pseudo keyword
-            STRICT_SETNAME = 74,
-            EXP = 75, // Exponentiation Operator
+            FIRST_BYTECODE_TOKEN = EOL + 1,
+            ENTERWITH = FIRST_BYTECODE_TOKEN,
+            LEAVEWITH = ENTERWITH + 1,
+            RETURN = LEAVEWITH + 1,
+            GOTO = RETURN + 1,
+            IFEQ = GOTO + 1,
+            IFNE = IFEQ + 1,
+            SETNAME = IFNE + 1,
+            BITOR = SETNAME + 1,
+            BITXOR = BITOR + 1,
+            BITAND = BITXOR + 1,
+            EQ = BITAND + 1,
+            NE = EQ + 1,
+            LT = NE + 1,
+            LE = LT + 1,
+            GT = LE + 1,
+            GE = GT + 1,
+            LSH = GE + 1,
+            RSH = LSH + 1,
+            URSH = RSH + 1,
+            ADD = URSH + 1,
+            SUB = ADD + 1,
+            MUL = SUB + 1,
+            DIV = MUL + 1,
+            MOD = DIV + 1,
+            NOT = MOD + 1,
+            BITNOT = NOT + 1,
+            POS = BITNOT + 1,
+            NEG = POS + 1,
+            NEW = NEG + 1,
+            DELPROP = NEW + 1,
+            TYPEOF = DELPROP + 1,
+            GETPROP = TYPEOF + 1,
+            GETPROPNOWARN = GETPROP + 1,
+            SETPROP = GETPROPNOWARN + 1,
+            GETELEM = SETPROP + 1,
+            SETELEM = GETELEM + 1,
+            CALL = SETELEM + 1,
+            NAME = CALL + 1,
+            NUMBER = NAME + 1,
+            STRING = NUMBER + 1,
+            NULL = STRING + 1,
+            THIS = NULL + 1,
+            FALSE = THIS + 1,
+            TRUE = FALSE + 1,
+            SHEQ = TRUE + 1, // shallow equality (===)
+            SHNE = SHEQ + 1, // shallow inequality (!==)
+            REGEXP = SHNE + 1,
+            BINDNAME = REGEXP + 1,
+            THROW = BINDNAME + 1,
+            RETHROW = THROW + 1, // rethrow caught exception: catch (e if ) use it
```

Comment:
When inserting new opcodes, only the line before and after the insertion point has to be changed